lib/bolt/inventory/options.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require 'bolt/config/options'
+
+module Bolt
+  class Inventory
+    module Options
+      # Top-level options available in the inventory.
+      OPTIONS = %w[
+        config
+        facts
+        features
+        groups
+        targets
+        vars
+      ].freeze
+
+      # Definitions used to validate the data.
+      # https://github.com/puppetlabs/bolt/blob/main/schemas/README.md
+      DEFINITIONS = {
```

Comment:
I think this needs to merge in the transport definitions as well, so that unknown keys in transport config are caught:
```
lucy ~/githubs/bolt ((b79def82...)) $ cat inventory.yaml 
---
groups:
  - name: docker
    targets:
      - uri: localhost:20022
      - uri: localhost:20023
      - uri: localhost:20024
    config:
      ssh:
        user: bolt
        ssl-verify: false
        foo: bar
lucy ~/githubs/bolt ((b79def82...)) $ be bolt command run whoami -t localhost
Started on localhost...
Finished on localhost:
  STDOUT:
    lucy
Successful on 1 target: localhost
Ran on 1 target in 0.01 sec
```

Reply:
The definitions are there, Bolt just wasn't logging the warnings on the validator. I swear I added that at some point, but guess not... anyway, it should warn now. :)